ext/session/tests/session_name_variation_null_byte.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Test session_name() function : null byte in session name
+--EXTENSIONS--
+session
+--SKIPIF--
+<?php include('skipif.inc'); ?>
+--FILE--
+<?php
+
+ob_start();
+
+var_dump(session_name("AB\0CD"));
+var_dump(session_start());
```

Comment:
I would just try-catch this and print the ValueError message rather than use the output buffering